### PR TITLE
(DOCSP-11372) Update Caps page

### PR DESCRIPTION
**Pull request by atsansone**
@andf-mongodb : [Staged](https://docs-mongodbcom-staging.corp.mongodb.com/meta/docsworker-xlarge/DOCSP-11372/style-guide/style/capitalization.html)

---
[Builds without errors on `style-guide-v3`.](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=5f209bc22f9a5cfb5b9c485d)

### CODE REVIEW

File: source/style-guide/style/capitalization.txt [hunk 1]
```diff
@@ -34,8 +40,10 @@ at the end of this topic.
    :depth: 1
    :class: singlecol
 
+.. _sg-cap-proper-nouns:
+
```
Comment (andf-mongodb) [inline source/style-guide/style/capitalization.txt:44]: I know there's a separate Titles and Headings page, but I think you need to talk about that here too. How about a tiny header, a one-liner "At MDB we capitalize titles and headers" and a link to the T&H style guide page? That should be all we need here.

Comment (andf-mongodb) [reply source/style-guide/style/capitalization.txt:44]: Ah, you should also include such a link in the table at the end of this topic that links out for related Capitalization concerns.

File: source/style-guide/style/capitalization.txt [hunk 2]
```diff
@@ -46,48 +54,38 @@ capitalization. Always follow the capitalization that's used by the
...
+   - Microsoft Windows
+   - MongoDB
+   - OpenStack
+   - PuTTY
+   - RackConnect
+   - Service Advertising Protocol
+   - SQL Server
+   - Wi-Fi
+   - WordPress
+
```
Comment (andf-mongodb) [inline source/style-guide/style/capitalization.txt:78]: Ready for more fun? Context-sensitive capitalization:

- "FTP" - the protocol, introduced [here](https://tools.ietf.org/html/rfc354) (and check out the gnarly capitalization of HOSTs on that RFC!)
- `ftp` - the command line FTP client, included on every *nix and macOS box.

Comment (andf-mongodb) [reply source/style-guide/style/capitalization.txt:78]: Also stellar inclusion of `PuTTY`, which highlights that there are no rules when it comes to naming your software (it's a play on the shell's TTY interface, but it totally loses its charm -- and ability to stand out in a crowd -- if just "Putty").

File: source/style-guide/style/capitalization.txt [hunk 3]
```diff
@@ -226,24 +224,30 @@ the terms.
...
 capitalized when used as common nouns. When in doubt, consult an
-existing style sheet, an editor, or the product team (but be aware that
-product teams sometimes tend to overcapitalize terms). Following are
-some tips to help you determine whether a noun should be capitalized:
+existing style sheet, an editor, or the product team.
+
+.. note:: Product teams sometimes tend to overcapitalize terms.
+
+Follow these tips to help you determine whether a noun should be
+capitalized:
```
Comment (andf-mongodb) [inline source/style-guide/style/capitalization.txt:250]: For me, this is all about: look it up in your current docs corpus. The hard thinking about this choice has already been done. My task, following, is merely to remain consistant.

File: source/style-guide/style/capitalization.txt (hunk 1 above)
Comment (andf-mongodb) [reply source/style-guide/style/capitalization.txt:44]: Still think we need a link to https://github.com/mongodb/docs-meta/pull/54 from somewhere on this page. Sorry if I missed it.

**Comment by atsansone** [comment]
@andf-mongodb : The link is on the page:

image

**Pull request merged**